sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.dto.ts

```diff
@@ -3,6 +3,7 @@ import {
   DesignationAgreementStatus,
   NOTE_DESCRIPTION_MAX_LENGTH,
 } from "@sims/sims-db";
+import { JsonMaxSize } from "../../../utilities/class-validation";
```

Comment:
Can you please add this to the index file and import it from there?

Reply:
It's added to class-validation's index. Do you mean import from the utilities` index?